#!/usr/bin/env python3
PACKAGE = "navigation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("dynamic_footprint",   bool_t,   0, "True if want to use dynamic footprint",  True)
gen.add("scaling_below", double_t, 0, "footprint scaling if below scaling velocity",    .01, -1,   1)
gen.add("scaling_over", double_t, 0, "footprint scaling if above scaling velocity",    .05, -1,   1)
gen.add("scaling_vel", double_t, 0, "threshold of velocity",    .2, 0,   100)
gen.add("drive_right", bool_t, 0, "True if want to drive mostly right",    True)

exit(gen.generate(PACKAGE, "navigation", "DynamicFootprint"))